File: scripts/generate_prisma.py
#!/usr/bin/env python3
import subprocess
import sys

print('[v0] Generating Prisma client...')
try:
    result = subprocess.run(
        ['npx', 'prisma', 'generate'],
        cwd='/vercel/share/v0-project',
        check=True,
    )
    print('[v0] ✓ Prisma client generated successfully!')
except subprocess.CalledProcessError as e:
    print(f'[v0] ✗ Error generating Prisma client')
    sys.exit(1)

File: scripts/generate-prisma.js
import { execSync } from 'child_process';
import { fileURLToPath } from 'url';
import path from 'path';

const __filename = fileURLToPath(import.meta.url);
const __dirname = path.dirname(__filename);
const projectRoot = path.join(__dirname, '..');

console.log('[v0] Generating Prisma client...');
try {
  execSync('npx prisma generate', {
    cwd: projectRoot,
    stdio: 'inherit',
  });
  console.log('[v0] Prisma client generated successfully!');
} catch (error) {
  console.error('[v0] Error generating Prisma client:', error.message);
  process.exit(1);
}
